plugin/trino-pinot/pom.xml

```diff
@@ -14,15 +14,32 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
-        <dep.pinot.version>0.12.1</dep.pinot.version>
+        <dep.pinot.version>1.1.0</dep.pinot.version>
+        <!--
+           Project's default for air.test.parallel is 'methods'. By design, 'instances' makes TestNG run tests from one class in a single thread.
+           As a side effect, it prevents TestNG from initializing multiple test instances upfront, which happens with 'methods'.
+           A potential downside can be long tail single-threaded execution of a single long test class.
+           TODO (https://github.com/trinodb/trino/issues/11294) remove when we upgrade to surefire with https://issues.apache.org/jira/browse/SUREFIRE-1967
+           -->
+        <air.test.parallel>instances</air.test.parallel>
+        <air.test.jvm.additional-arguments>${air.test.jvm.additional-arguments.default}
```

Comment:
Are these arguments required not only in tests but also in actual Trino clusters?

Reply:
Yes